### PR TITLE
Job Integration Tests

### LINKED ISSUE #91: Integration Tests for Jobs

### PR DESCRIPTION
**Pull request by jaredwebber**
## Tickets
Closes #91 
## Summary Of Changes
- Added Integration tests for Job
  - Updated required Exception checks
## Notes for Reviewers
- Built on top of #119

### CODE REVIEW

**Review by LokiFrostGiant — APPROVED** [review]
Looks good, no obvious mistakes or missing tests. Let's see what one other person says.

File: backend/tests/integration/test_job.py [hunk 1]
```diff
@@ -0,0 +1,511 @@
...
+            content_type="application/json",
+        )
+
+        jobs = [{"id": 1, "kcolumn": self.column.id, "position_title": "Developer", "company": "Google", "type": None}]
+
+        # Check the response
+        self.assertEqual(response.status_code, 200)
+        self.assertEqual(response.content.decode("utf-8"), json.dumps({"jobs": jobs}))
+
+    def test_get_minimum_jobs_nonexistant_user(self):
```
Comment (drkitt) [inline backend/tests/integration/test_job.py:379]: 🤓 Minor spelling mistake
```suggestion
    def test_get_minimum_jobs_nonexistent_user(self):
```

**Review by drkitt — COMMENTED** [review]
The tests weren't running for me until I moved them into the `backend/job/tests` folder. Is it like that on your end too?

**Comment by jaredwebber** [comment]
> The tests weren't running for me until I moved them into the `backend/job/tests` folder. Is it like that on your end too?

@drkitt you'll have to run `manage.py test tests/integration/` - this lets us have a separate GitHub action to trigger unit vs integration tests

**Review by drkitt — APPROVED** [review]

**Pull request merged**